Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -890,7 +895,11 @@ def _exec_editor_multitest(self, request: Request, workspace: AbstractWorkspace,
                 results[test_spec_name] = Result.Timeout.create(timed_out_result.test_spec,
                                                                 results[test_spec_name].output,
                                                                 self.timeout_editor_shared_test, result.editor_log)
-
+        finally:
+            log_folder = editor_utils.retrieve_log_path(run_id, workspace)
```

Comment:
In the parallel-editors case, this results in a separate folder per editor, correct?

Reply:
Instead of moving their contents after execution, perhaps these folders' initial location can be rerouted directly into /<cmake_build>/testing/, following the pattern used by the artifact manager?

Reply:
Fixed as suggested